renderer/pages/index.js

```diff
@@ -21,6 +21,8 @@ import Save from '../components/save-button'
 import Open from '../components/open-button'
 import New from '../components/new-button'
 import Export from '../components/export-button'
+import Create from '../components/window-button'
+import imageDrop from '../components/image-drop'
```

Comment:
Aside of that file not being a component it's only used inside this page so it doesn't need to be externalized